app/rtc_compare.py

```diff
@@ -629,9 +646,12 @@ def compare_rtc_s1_products(file_1, file_2):
                   f' ({image_1.dtype}) vs. ({image_2.dtype})\n')
             return False
 
-        flag_bands_are_equal = np.allclose(
+        failed_pixel_index = ~np.isclose(
```

Comment:
Thank you for these updates, @seongsujeong . Are you changing this because the error or a few pixels was too large and we can't use overall relative and abs. error tolerances?

Reply:
Yes. The rationale for this is that very few pixels have larger error than the tolerance we've set. In this code I've set the ratio as 0.002%.

Reply:
Do you have some values? I'm asking because I'm not sure if that's really acceptable to have large errors even if it's in a few pixels.

Reply:
# of the pixels in the band in the raster: 5,088,741
# of the failed pixel: 96

% of the failed pixels: 0.0019%

# Maximum difference of the pixel value: 0.00024414062
1st image at the maximum difference: 0.32885742
2nd imag at the maximum differencee: 0.32861328

Reply:
And below is a toy code to briefly demonstrate how much error takes place between single precision and half precision:

```python
a=np.arange(0, 1, 1.0e-6).astype(np.float32)
b=np.arange(0, 1, 1.0e-6).astype(np.float16)
plt.plot(a, b - a)
```
![Screenshot 2023-05-22 at 10 35 00](https://github.com/gshiroma/RTC/assets/27862199/22eba61c-e81d-42d7-ac39-3e02227ab35f)

Reply:
Thank you for these numbers, @seongsujeong . If I'm not mistaken, the products should both be generated with Float32 half-precision precision. I'm going to slack you to discuss this.